scripts/src/owners/owners_file.py

```diff
@@ -62,9 +62,19 @@ def get_chart(owner_data):
 
 
 def get_web_catalog_only(owner_data):
-    return owner_data.get("web_catalog_only", False) or owner_data.get(
-        "providerDelivery", False
-    )
+    success = False
+    owners_web_catalog_only = False
+    owners_providerDelivery = False
+
+    with contextlib.suppress(KeyError):
+        owners_web_catalog_only = owner_data["web_catalog_only"]
+        success = True
+
+    with contextlib.suppress(KeyError):
+        owners_providerDelivery = owner_data["providerDelivery"]
+        success = True
+
+    return success, owners_web_catalog_only or owners_providerDelivery
```

Comment:
Bit of a preference, but I'd like to see the result order inverted such that the answer to the function call "get_web_catalog_only" is the first returned parameter, and whether or not the key was actually found, or the returned value was a fallback false is the second. Similar to what you'd see in Go: 

```go
if theValue, exists := mymap["foo"]; exists {
    ...
}
```

Reply:
Also, `success` isn't as meaningful as I'd like. Better to have `keyFound` or something that tells us clearly what was returned.